### DISCUSSION TITLE
SailPoint Developer Community

### DISCUSSION THREAD

**Discussion by jordan-violet-sp** [facebook/docusaurus#8144] [Show and tell]
Site title: SailPoint Developer Community
Site description: A community for developers on SailPoint's identity security platforms.
Site URL: developer.sailpoint.com
Site source code URL: https://github.com/sailpoint-oss/developer.sailpoint.com
Tags: opensource, product, design

**Comment by slorber** [comment]
Thanks for the submission

You are supposed to comment on the initial issue, so I added the comment for you here: https://github.com/facebook/docusaurus/discussions/7826#discussioncomment-3763453

Why did you think you should open a new discussion? how can we prevent this from happening again?

**Reply by jordan-violet-sp** [reply]
Simply incorrectly interpreting the submission guideline! I interpreted the "Just post a comment in this GitHub discussion..." as posting a new discussion topic. I of course now read it differently knowing the solution.

I suppose if it read as "Post a comment below..." it would have been more clear (than it already was--my mistake there!). Thank you for adding the comment for me, though!

**Reply by slorber** [reply]
Thanks, words matter :)

Modified to 

> Just post a comment below in **this** GitHub discussion with some basic information:
